### DISCUSSION TITLE
Live Streaming using 'react-native-video'

### DISCUSSION THREAD

**Discussion by singhroshan18** [TheWidlarzGroup/react-native-video#3939] [Q&A]
### What are you trying to do?

Implement live streaming in a React Native application using `react-native-video`. Looking for guidance on how to set this up.

**Comment by freeboub** [comment]
check the sample app, it includes some live samples

**Reply by singhroshan18** [reply]
where can I find the sample app?

does it support 'rtmp' streaming ?

**Reply by freeboub** [reply]
rtmp not supported: https://github.com/TheWidlarzGroup/react-native-video/issues/667 (notice that it will not be available on ios due to avplayer limitation)
for the sample app: https://thewidlarzgroup.github.io/react-native-video/other/debug
